### PR TITLE
Retrieving profile pic

### PR DESCRIPTION
**Pull request by Mounir-mk**

### CODE REVIEW

File: backend/src/controllers/userControllers.js [hunk 1]
```diff
@@ -9,18 +9,15 @@ const uploadFile = upload.single("avatar");
...
-        req.body.profilePicture = path;
-        next();
+    const modifiedName = `${uuidv4()}-${originalname}`;
+    const path = `uploads/${modifiedName}`;
+    fs.rename(`uploads/${filename}`, path, (err) => {
+      if (err) {
+        res.sendStatus(500);
       }
-    );
+      req.body.image_url = `http://localhost:5000/${modifiedName}`;
```
Comment (rocambille) [inline backend/src/controllers/userControllers.js:18]: only store the path here, remove the scheme and the domain name

File: frontend/src/pages/Dashboard.jsx [hunk 2]
```diff
@@ -32,6 +32,7 @@ function Dashboard() {
             .map((decision) => (
               <DashboardCard
                 key={decision.id}
+                avatar={decision.image_url}
```
Comment (rocambille) [inline frontend/src/pages/Dashboard.jsx:35]: prefix with backend url from the .env

File: frontend/src/pages/Dashboard.jsx [hunk 3]
```diff
@@ -49,6 +50,7 @@ function Dashboard() {
...
```
Comment (rocambille) [inline frontend/src/pages/Dashboard.jsx:53]: prefix with backend url from the .env

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by Mounir-mk** [comment]
I took advantage of this change request to modify some things in the application behavior

**Pull request merged**